x-pack/plugins/security/server/index.ts

```diff
@@ -50,3 +50,5 @@ export const plugin: PluginInitializer<
   RecursiveReadonly<SecurityPluginStart>,
   PluginSetupDependencies
 > = (initializerContext: PluginInitializerContext) => new SecurityPlugin(initializerContext);
+
+export type { CasesSupportedOperations } from '../server/authorization/privileges/feature_privilege_builder';
```

Comment:
Not sure if this is the best place to expose this. Another idea I had was to move the array of strings to the `common` folder and have `x-pack/plugins/security/server/authorization/privileges/feature_privilege_builder/cases.ts` import them from `common` and then define the `CasesSupportedOperations` type in the `common` folder.

Reply:
If it's only being used in server-side code, then the `server` folder is the right place for it :+1:

That said, our convention is to [import (and export) only top-level modules](https://github.com/elastic/kibana/blob/30872e9063d4171f51e8f950125977ea006aaf4d/STYLEGUIDE.mdx#import-only-top-level-modules), so it would be great if you could remove this line and export this type along with the rest of the `./authorization` module above on line 31 🙌

Reply:
Oh good point!